lib/menubar/init.lua.in

```diff
@@ -280,9 +321,6 @@ end
 --- Get a menubar wibox.
 -- @return menubar wibox.
 function menubar.get()
-    if app_folders then
```

Comment:
It was never used in awesome itself, and was probably used to override / provide it from outside.

Given this, it should probably be used via `menubar.app_folders`, but maybe only when it's being requested again..